app/views/search/_filters.html.erb

```diff
@@ -1,32 +1,32 @@
 <h2 class="heading-medium"> <%= t('.filter_by') %></h2>
 
-<div class="form-group">
-  <%= label_tag 'publisher', t('.filter_by_publisher'), class: 'form-label' %>
-  <%= select_tag('filters[publisher]', options_for_select(dataset_publishers_for_select, selected_publisher), id: 'publisher', class: 'form-control dgu-filters__filter', include_blank: true) %>
+<div class="govuk-form-group">
+  <%= label_tag 'publisher', t('.filter_by_publisher'), class: 'govuk-label' %>
```

Comment:
If DGU uses it, the [GDS Rubocop rules](https://gds-way.cloudapps.digital/manuals/programming-languages/ruby.html#code-formatting) ask for double quotes. 

It can be run to automatically fix some things, so these might be better in another pull request.
```suggestion
  <%= label_tag "publisher", t(".filter_by_publisher"), class: "govuk-label" %>
```